packages/sdk/src/Actions/EventAction.ts

```diff
@@ -1242,3 +1268,38 @@ export function anyActionParameter(): Criteria {
     filterData: zeroHash,
   };
 }
+
+/**
+ * Creates an ActionClaimant object that represents the msg.sender as the claimant.
+ * This function is useful when you want to set up an action where the transaction sender
+ * (msg.sender in Solidity terms) is always considered the valid claimant, regardless of
```

Comment:
I think we need to be very careful here. Are we referring to msg.sender (which can exist at multiple layers when internal transaction are present, or the [`from`](https://github.com/wevm/viem/blob/main/src/types/transaction.ts#L59) param in a transaction/ transaction receipt? In solidity this is technically the [tx.origin](https://ethereum.stackexchange.com/questions/1891/whats-the-difference-between-msg-sender-and-tx-origin)

Reply:
Explicitly we should refer to this as `from` - we don't want the `msg.sender` confusion occurring in any external facing docs/comments